### PR TITLE
Bomberman

### PR DESCRIPTION
**Pull request by MasterpieceElbow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+
+There are `player` (green rectangle) and `walls` (white squares).
+You should implement these features:
+- Player must be able to move up, down, left, right when user
+presses arrows (overwrite method update).
+- Player cannot walk away from field and walk over the walls.
+- Player should deploy a bomb when user presses space key. Player cannot
+put bomb if less than 1 second has passed from last deploying (that is
+all so far, bomb will explode later). Player cannot walk over the bomb also.
+- Replace simple colored rectangles with pictures given in `"app/images/`.
```
[the remark this answers is not in the document]
Comment (IvanRamyk) [reply README.md:19]: Is there information about how to change the sprite image based on the direction?

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,50 @@
-# Python boilerplate for GitHub tasks
+# Bomberman pygame application day 1
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+- [Pygame primer](https://realpython.com/pygame-a-primer/)
+- [Pygame sprites and groups](https://kidscancode.org/blog/2016/08/pygame_1-2_working-with-sprites/)
+
+Games! Games is perfect example to show OOP principles at work.
```
Comment (IvanRamyk) [inline README.md:7]:
```suggestion
Games! Game is a perfect example to show OOP principles at work.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,50 @@
...
+You could play game Bomberman in your youth or childhood, now
+you have to write your own a little modified version.
```
Comment (IvanRamyk) [inline README.md:9]:
```suggestion
it's time to write your own a little modified version.
```
Sounds more friendly for me

File: README.md [hunk 4]
```diff
@@ -1,4 +1,50 @@
...
+You already have initial structure of game Bomberman.
```
Comment (IvanRamyk) [inline README.md:10]:
```suggestion
You already have the initial structure of the game Bomberman.
```

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
I'd recommend using Grammarly to check simple grammar mistakes

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MasterpieceElbow) [reply README.md]: It is already changed. 0_o

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 5]
```diff
@@ -1,3 +1,96 @@
...
+        self.rect = self.surf.get_rect()
+
+    def update(self):
+        pass
+
+
+class Wall(pygame.sprite.Sprite):
+    def __init__(self, center_pos: tuple):
+        super().__init__()
+        self.width = 50
```
[the remark this answers is not in the document]
Comment (MasterpieceElbow) [reply app/main.py]: @IvanRamyk Done

**Pull request merged**